diff --git a/node_modules/@apollo/client/core/ObservableQuery.js b/node_modules/@apollo/client/core/ObservableQuery.js
index fd24c61..91debf3 100644
--- a/node_modules/@apollo/client/core/ObservableQuery.js
+++ b/node_modules/@apollo/client/core/ObservableQuery.js
@@ -95,6 +95,7 @@ var ObservableQuery = (function (_super) {
         }
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
diff --git a/node_modules/@apollo/client/core/core.cjs b/node_modules/@apollo/client/core/core.cjs
index 86f7d1e..e5b6f6d 100644
--- a/node_modules/@apollo/client/core/core.cjs
+++ b/node_modules/@apollo/client/core/core.cjs
@@ -215,6 +215,7 @@ var ObservableQuery = (function (_super) {
             this.queryManager.transform(this.options.query).hasForcedResolvers) ;
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
