diff --git a/node_modules/@apollo/client/apollo-client.cjs b/node_modules/@apollo/client/apollo-client.cjs
index 6e3c874..7497ee8 100644
--- a/node_modules/@apollo/client/apollo-client.cjs
+++ b/node_modules/@apollo/client/apollo-client.cjs
@@ -4742,6 +4742,7 @@ var ObservableQuery = (function (_super) {
         }
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
diff --git a/node_modules/@apollo/client/core/ObservableQuery.js b/node_modules/@apollo/client/core/ObservableQuery.js
index 82df343..5b2b3df 100644
--- a/node_modules/@apollo/client/core/ObservableQuery.js
+++ b/node_modules/@apollo/client/core/ObservableQuery.js
@@ -100,6 +100,7 @@ var ObservableQuery = (function (_super) {
         }
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
diff --git a/node_modules/@apollo/client/core/core.cjs b/node_modules/@apollo/client/core/core.cjs
index 810f46c..701e7cb 100644
--- a/node_modules/@apollo/client/core/core.cjs
+++ b/node_modules/@apollo/client/core/core.cjs
@@ -286,6 +286,7 @@ var ObservableQuery = (function (_super) {
         }
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
