diff --git a/node_modules/@apollo/client/__cjs/core/ObservableQuery.cjs b/node_modules/@apollo/client/__cjs/core/ObservableQuery.cjs
index 36bf305..bb001d9 100644
--- a/node_modules/@apollo/client/__cjs/core/ObservableQuery.cjs
+++ b/node_modules/@apollo/client/__cjs/core/ObservableQuery.cjs
@@ -234,6 +234,7 @@ class ObservableQuery {
                 diff.result ?? undefined
                 : undefined;
             return this.maskResult({
+                diff,
                 data,
                 dataState: diff.complete ? "complete"
                     : data === undefined ? "empty"
diff --git a/node_modules/@apollo/client/core/ObservableQuery.js b/node_modules/@apollo/client/core/ObservableQuery.js
index 8c68f00..9b746e3 100644
--- a/node_modules/@apollo/client/core/ObservableQuery.js
+++ b/node_modules/@apollo/client/core/ObservableQuery.js
@@ -226,6 +226,7 @@ export class ObservableQuery {
                 diff.result ?? undefined
                 : undefined;
             return this.maskResult({
+                diff,
                 data,
                 dataState: diff.complete ? "complete"
                     : data === undefined ? "empty"
