diff --git a/node_modules/@apollo/client/apollo-client.cjs b/node_modules/@apollo/client/apollo-client.cjs
index 17856d7..7feed7c 100644
--- a/node_modules/@apollo/client/apollo-client.cjs
+++ b/node_modules/@apollo/client/apollo-client.cjs
@@ -4935,6 +4935,7 @@ var ObservableQuery =  (function (_super) {
         }
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
diff --git a/node_modules/@apollo/client/core/ObservableQuery.js b/node_modules/@apollo/client/core/ObservableQuery.js
index b8339a6..18b8ab1 100644
--- a/node_modules/@apollo/client/core/ObservableQuery.js
+++ b/node_modules/@apollo/client/core/ObservableQuery.js
@@ -158,6 +158,7 @@ var ObservableQuery = /** @class */ (function (_super) {
         }
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
diff --git a/node_modules/@apollo/client/core/core.cjs b/node_modules/@apollo/client/core/core.cjs
index f15ae51..26648bd 100644
--- a/node_modules/@apollo/client/core/core.cjs
+++ b/node_modules/@apollo/client/core/core.cjs
@@ -294,6 +294,7 @@ var ObservableQuery =  (function (_super) {
         }
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
diff --git a/node_modules/@apollo/client/core/core.cjs.native.js b/node_modules/@apollo/client/core/core.cjs.native.js
index f15ae51..26648bd 100644
--- a/node_modules/@apollo/client/core/core.cjs.native.js
+++ b/node_modules/@apollo/client/core/core.cjs.native.js
@@ -294,6 +294,7 @@ var ObservableQuery =  (function (_super) {
         }
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
