diff --git a/node_modules/@apollo/client/core/ObservableQuery.js b/node_modules/@apollo/client/core/ObservableQuery.js
index 627daf6..baa37f7 100644
--- a/node_modules/@apollo/client/core/ObservableQuery.js
+++ b/node_modules/@apollo/client/core/ObservableQuery.js
@@ -95,6 +95,7 @@ var ObservableQuery = (function (_super) {
         }
         else {
             var diff = this.queryInfo.getDiff();
+            result.diff = diff;
             if (diff.complete || this.options.returnPartialData) {
                 result.data = diff.result;
             }
