diff --git a/node_modules/@chakra-ui/button/dist/cjs/button.js b/node_modules/@chakra-ui/button/dist/cjs/button.js
index 970ab9d..9d773ea 100644
--- a/node_modules/@chakra-ui/button/dist/cjs/button.js
+++ b/node_modules/@chakra-ui/button/dist/cjs/button.js
@@ -88,7 +88,7 @@ var Button = /*#__PURE__*/(0, _system.forwardRef)(function (props, ref) {
     "data-loading": (0, _utils.dataAttr)(isLoading),
     __css: buttonStyles,
     className: (0, _utils.cx)("chakra-button", className)
-  }, rest), leftIcon && !isLoading && /*#__PURE__*/React.createElement(ButtonIcon, {
+  }, _utils.filterUndefined(rest)), leftIcon && !isLoading && /*#__PURE__*/React.createElement(ButtonIcon, {
     marginEnd: iconSpacing
   }, leftIcon), isLoading && spinnerPlacement === "start" && /*#__PURE__*/React.createElement(ButtonSpinner, {
     className: "chakra-button__spinner--start",
diff --git a/node_modules/@chakra-ui/button/dist/esm/button.js b/node_modules/@chakra-ui/button/dist/esm/button.js
index d4c70f2..50d4ec6 100644
--- a/node_modules/@chakra-ui/button/dist/esm/button.js
+++ b/node_modules/@chakra-ui/button/dist/esm/button.js
@@ -5,7 +5,7 @@ function _extends() { _extends = Object.assign || function (target) { for (var i
 import { mergeRefs } from "@chakra-ui/react-utils";
 import { Spinner } from "@chakra-ui/spinner";
 import { chakra, forwardRef, omitThemingProps, useStyleConfig } from "@chakra-ui/system";
-import { cx, dataAttr, mergeWith, __DEV__ } from "@chakra-ui/utils";
+import { cx, dataAttr, mergeWith, __DEV__, filterUndefined } from "@chakra-ui/utils";
 import * as React from "react";
 import { useButtonGroup } from "./button-group";
 export var Button = /*#__PURE__*/forwardRef((props, ref) => {
@@ -72,7 +72,7 @@ export var Button = /*#__PURE__*/forwardRef((props, ref) => {
     "data-loading": dataAttr(isLoading),
     __css: buttonStyles,
     className: cx("chakra-button", className)
-  }, rest), leftIcon && !isLoading && /*#__PURE__*/React.createElement(ButtonIcon, {
+  }, filterUndefined(rest)), leftIcon && !isLoading && /*#__PURE__*/React.createElement(ButtonIcon, {
     marginEnd: iconSpacing
   }, leftIcon), isLoading && spinnerPlacement === "start" && /*#__PURE__*/React.createElement(ButtonSpinner, {
     className: "chakra-button__spinner--start",
