diff --git a/node_modules/@faire/mjml-react/esm/utils/render.js b/node_modules/@faire/mjml-react/esm/utils/render.js
index 45f1e29..bee4a1a 100644
--- a/node_modules/@faire/mjml-react/esm/utils/render.js
+++ b/node_modules/@faire/mjml-react/esm/utils/render.js
@@ -1,4 +1,4 @@
-import { minify as htmlMinify } from "html-minifier";
+// import { minify as htmlMinify } from "html-minifier";
 import mjml2html from "mjml";
 import { renderToMjml } from "./renderToMjml";
 export function render(email, options = {}) {
diff --git a/node_modules/@faire/mjml-react/utils/render.js b/node_modules/@faire/mjml-react/utils/render.js
index ec6a60a..0e9e030 100644
--- a/node_modules/@faire/mjml-react/utils/render.js
+++ b/node_modules/@faire/mjml-react/utils/render.js
@@ -4,7 +4,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.render = void 0;
-const html_minifier_1 = require("html-minifier");
+// const html_minifier_1 = require("html-minifier");
 const mjml_1 = __importDefault(require("mjml"));
 const renderToMjml_1 = require("./renderToMjml");
 function render(email, options = {}) {
