diff --git a/node_modules/@formatjs/intl/lib/src/create-intl.js b/node_modules/@formatjs/intl/lib/src/create-intl.js
index ae12493..2334286 100755
--- a/node_modules/@formatjs/intl/lib/src/create-intl.js
+++ b/node_modules/@formatjs/intl/lib/src/create-intl.js
@@ -15,7 +15,7 @@ function messagesContainString(messages) {
 function verifyConfigMessages(config) {
     if (config.defaultRichTextElements &&
         messagesContainString(config.messages || {})) {
-        console.warn("[@formatjs/intl] \"defaultRichTextElements\" was specified but \"message\" was not pre-compiled. \nPlease consider using \"@formatjs/cli\" to pre-compile your messages for performance.\nFor more details see https://formatjs.io/docs/getting-started/message-distribution");
+        // console.warn("[@formatjs/intl] \"defaultRichTextElements\" was specified but \"message\" was not pre-compiled. \nPlease consider using \"@formatjs/cli\" to pre-compile your messages for performance.\nFor more details see https://formatjs.io/docs/getting-started/message-distribution");
     }
 }
 /**
diff --git a/node_modules/@formatjs/intl/src/create-intl.js b/node_modules/@formatjs/intl/src/create-intl.js
index 5b1821b..2975f11 100755
--- a/node_modules/@formatjs/intl/src/create-intl.js
+++ b/node_modules/@formatjs/intl/src/create-intl.js
@@ -18,7 +18,7 @@ function messagesContainString(messages) {
 function verifyConfigMessages(config) {
     if (config.defaultRichTextElements &&
         messagesContainString(config.messages || {})) {
-        console.warn("[@formatjs/intl] \"defaultRichTextElements\" was specified but \"message\" was not pre-compiled. \nPlease consider using \"@formatjs/cli\" to pre-compile your messages for performance.\nFor more details see https://formatjs.io/docs/getting-started/message-distribution");
+        // console.warn("[@formatjs/intl] \"defaultRichTextElements\" was specified but \"message\" was not pre-compiled. \nPlease consider using \"@formatjs/cli\" to pre-compile your messages for performance.\nFor more details see https://formatjs.io/docs/getting-started/message-distribution");
     }
 }
 /**
