diff --git a/node_modules/@types/intercom-web/index.d.ts b/node_modules/@types/intercom-web/index.d.ts
index 06afb25..5176bdf 100755
--- a/node_modules/@types/intercom-web/index.d.ts
+++ b/node_modules/@types/intercom-web/index.d.ts
@@ -62,6 +62,7 @@ declare namespace Intercom_ {
         trackEvent: (tag?: string, metadata?: any) => void;
         getVisitorId: () => string;
         startTour: (tourId: number) => void;
+        showArticle: (articleId: number) => void;
     }
 
     type IntercomCommand = keyof IntercomCommandSignature;
