diff --git a/node_modules/@types/segment-analytics/index.d.ts b/node_modules/@types/segment-analytics/index.d.ts
index 1be4a82..220121a 100644
--- a/node_modules/@types/segment-analytics/index.d.ts
+++ b/node_modules/@types/segment-analytics/index.d.ts
@@ -192,9 +192,6 @@ declare namespace SegmentAnalytics {
     }
 }
 
-declare var analytics: SegmentAnalytics.AnalyticsJS;
-
-declare module "@segment/analytics.js-core" {
-    var analytics: SegmentAnalytics.AnalyticsJS;
-    export default analytics;
+declare interface Window {
+    analytics: SegmentAnalytics.AnalyticsJS | undefined;
 }
