diff --git a/node_modules/@udecode/plate-core/dist/index.es.js b/node_modules/@udecode/plate-core/dist/index.es.js
index 2a95c1a..65f4acc 100644
--- a/node_modules/@udecode/plate-core/dist/index.es.js
+++ b/node_modules/@udecode/plate-core/dist/index.es.js
@@ -8515,7 +8515,8 @@ const useSlateProps = ({
       key: editor.key,
       editor,
       onChange,
-      value
+      value,
+      initialValue: value,
     };
   }, [editor, onChange, value]);
 };
diff --git a/node_modules/@udecode/plate-core/dist/index.js b/node_modules/@udecode/plate-core/dist/index.js
index 0ca8fa1..24e7f04 100644
--- a/node_modules/@udecode/plate-core/dist/index.js
+++ b/node_modules/@udecode/plate-core/dist/index.js
@@ -8521,7 +8521,8 @@ const useSlateProps = ({
       key: editor.key,
       editor,
       onChange,
-      value
+      value,
+      initialValue: value,
     };
   }, [editor, onChange, value]);
 };
