diff --git a/node_modules/@udecode/plate-mention/dist/index.es.js b/node_modules/@udecode/plate-mention/dist/index.es.js
index a816045..c4fa3d8 100644
--- a/node_modules/@udecode/plate-mention/dist/index.es.js
+++ b/node_modules/@udecode/plate-mention/dist/index.es.js
@@ -577,7 +577,7 @@ const getMentionOnSelectItem = ({
     });
     select(editor, targetRange);
     withoutMergingHistory(editor, () => removeNodes(editor, {
-      match: node => node.type === ELEMENT_MENTION_INPUT
+      match: node => isNodeMentionInput(editor, node)
     }));
     insertNodes(editor, {
       type,
diff --git a/node_modules/@udecode/plate-mention/dist/index.js b/node_modules/@udecode/plate-mention/dist/index.js
index 726baf9..4e17212 100644
--- a/node_modules/@udecode/plate-mention/dist/index.js
+++ b/node_modules/@udecode/plate-mention/dist/index.js
@@ -581,7 +581,7 @@ const getMentionOnSelectItem = ({
     });
     plateCommon.select(editor, targetRange);
     plateCommon.withoutMergingHistory(editor, () => plateCommon.removeNodes(editor, {
-      match: node => node.type === ELEMENT_MENTION_INPUT
+      match: node => isNodeMentionInput(editor, node)
     }));
     plateCommon.insertNodes(editor, {
       type,
