diff --git a/node_modules/@udecode/plate-core/dist/index.es.js b/node_modules/@udecode/plate-core/dist/index.es.js
index d8b2ea8..f2bbaf6 100644
--- a/node_modules/@udecode/plate-core/dist/index.es.js
+++ b/node_modules/@udecode/plate-core/dist/index.es.js
@@ -8734,14 +8734,14 @@ const withInlineVoid = editor => {
   const inlineTypes = [];
   const voidTypes = [];
   editor.plugins.forEach(plugin => {
-    if (!plugin.key) return;
+    if (!plugin.type) return;
 
     if (plugin.isInline) {
-      inlineTypes.push(plugin.key);
+      inlineTypes.push(plugin.type);
     }
 
     if (plugin.isVoid) {
-      voidTypes.push(plugin.key);
+      voidTypes.push(plugin.type);
     }
   });
 
