diff --git a/node_modules/eslint-plugin-formatjs/rules/no-literal-string-in-jsx.js b/node_modules/eslint-plugin-formatjs/rules/no-literal-string-in-jsx.js
index 9d5c5c7..7a4f90b 100644
--- a/node_modules/eslint-plugin-formatjs/rules/no-literal-string-in-jsx.js
+++ b/node_modules/eslint-plugin-formatjs/rules/no-literal-string-in-jsx.js
@@ -15,6 +15,7 @@ const defaultPropIncludePattern = [
     ['[a-z]*([a-z0-9])', '(placeholder|title)'],
     ['img', 'alt'],
 ];
+const IGNORE_REGEXES = [/^[^\p{L}]*$/u, /^parallel$/i, /^dow jones$/i];
 const defaultPropExcludePattern = [];
 function stringifyJsxTagName(tagName) {
     switch (tagName.type) {
@@ -102,11 +103,16 @@ const rule = {
         const checkJSXExpression = (node) => {
             // Check if this is either a string literal / template literal, or the concat of them.
             // It also ignores the empty string.
-            if ((node.type === 'Literal' &&
+            if (node.type === 'Literal' &&
                 typeof node.value === 'string' &&
-                node.value.length > 0) ||
-                (node.type === 'TemplateLiteral' &&
-                    (node.quasis.length > 1 || node.quasis[0].value.raw.length > 0))) {
+                !IGNORE_REGEXES.some(re => re.test(node.value))) {
+                context.report({
+                    node: node,
+                    message: 'Cannot have untranslated text in JSX',
+                });
+            }
+            else if (node.type === 'TemplateLiteral' &&
+                     node.quasis.some(q => !IGNORE_REGEXES.some(re => re.test(q.value.raw)))) {
                 context.report({
                     node: node,
                     message: 'Cannot have untranslated text in JSX',
@@ -147,7 +153,7 @@ const rule = {
                 }
                 if (node.value.type === 'Literal' &&
                     typeof node.value.value === 'string' &&
-                    node.value.value.length > 0) {
+                    !IGNORE_REGEXES.some(re => re.test(node.value.value))) {
                     context.report({
                         node: node,
                         message: 'Cannot have untranslated text in JSX',
@@ -163,6 +169,10 @@ const rule = {
                 if (!node.value.replace(/\s*/gm, '')) {
                     return;
                 }
+                // Ignore symbols
+                if (IGNORE_REGEXES.some(re => re.test(node.value))) {
+                    return;
+                }
                 context.report({
                     node: node,
                     message: 'Cannot have untranslated text in JSX',
