diff --git a/node_modules/exceljs/index.d.ts b/node_modules/exceljs/index.d.ts
index 9a37487..8df9dc2 100755
--- a/node_modules/exceljs/index.d.ts
+++ b/node_modules/exceljs/index.d.ts
@@ -1173,7 +1173,7 @@ export interface Worksheet {
 	 * Note: these column structures are a workbook-building convenience only,
 	 * apart from the column width, they will not be fully persisted.
 	 */
-	columns: Array<Column>;
+	columns: Array<Partial<Column>>;
 
 	/**
 	 * The total row size of the document. Equal to the row number of the last row that has values.
