diff --git a/node_modules/htmlnano/lib/helpers.cjs b/node_modules/htmlnano/lib/helpers.cjs
index 27d9858..6a6fef4 100644
--- a/node_modules/htmlnano/lib/helpers.cjs
+++ b/node_modules/htmlnano/lib/helpers.cjs
@@ -52,7 +52,8 @@ function isComment(content) {
   return false;
 }
 function isConditionalComment(content) {
-  return (content || '').trim().startsWith('<!--[if');
+  const clean = (content || '').trim(); 
+  return clean.startsWith('<!--[if') || clean === '<!--<![endif]-->';
 }
 function isStyleNode(node) {
   return node.tag === 'style' && !isAmpBoilerplate(node) && 'content' in node && node.content.length > 0;
diff --git a/node_modules/htmlnano/lib/helpers.mjs b/node_modules/htmlnano/lib/helpers.mjs
index 07f0a43..00d9b48 100644
--- a/node_modules/htmlnano/lib/helpers.mjs
+++ b/node_modules/htmlnano/lib/helpers.mjs
@@ -24,7 +24,8 @@ export function isComment(content) {
 }
 
 export function isConditionalComment(content) {
-    return (content || '').trim().startsWith('<!--[if');
+    const clean = (content || '').trim(); 
+    return clean.startsWith('<!--[if') || clean === '<!--<![endif]-->';
 }
 
 export function isStyleNode(node) {
