diff --git a/node_modules/imask/index.d.ts b/node_modules/imask/index.d.ts
index af4e9ef..a5b87a6 100644
--- a/node_modules/imask/index.d.ts
+++ b/node_modules/imask/index.d.ts
@@ -187,7 +187,7 @@ declare namespace IMask {
       doCommit(): void;
       nearestInputPos(cursorPos: number, direction: Direction): number;
     }
-    type PatternDefinition = AnyMasked | AnyMaskedOptions & Pick<
+    type PatternDefinition = RegExp | AnyMasked | AnyMaskedOptions & Pick<
       PatternInputDefinition,
       'lazy' | 'placeholderChar' | 'displayChar'
     >;
