diff --git a/node_modules/liquidjs/dist/liquid.browser.esm.js b/node_modules/liquidjs/dist/liquid.browser.esm.js
index b69a940..23b36a3 100644
--- a/node_modules/liquidjs/dist/liquid.browser.esm.js
+++ b/node_modules/liquidjs/dist/liquid.browser.esm.js
@@ -1299,8 +1299,11 @@ const defaultOperators = {
     },
     'contains': (l, r) => {
         l = toValue(l);
-        r = toValue(r);
-        return l && isFunction(l.indexOf) ? l.indexOf(r) > -1 : false;
+        if (l && isArray(l))
+            return l.some((i) => equal(i, r));
+        if (l && isString(l))
+            return l.indexOf(toValue(r)) > -1;
+        return false;
     },
     'not': (v, ctx) => isFalsy(toValue(v), ctx),
     'and': (l, r, ctx) => isTruthy(toValue(l), ctx) && isTruthy(toValue(r), ctx),
diff --git a/node_modules/liquidjs/dist/liquid.node.cjs.js b/node_modules/liquidjs/dist/liquid.node.cjs.js
index 95e7262..eed4d33 100644
--- a/node_modules/liquidjs/dist/liquid.node.cjs.js
+++ b/node_modules/liquidjs/dist/liquid.node.cjs.js
@@ -1286,8 +1286,11 @@ const defaultOperators = {
     },
     'contains': (l, r) => {
         l = toValue(l);
-        r = toValue(r);
-        return l && isFunction(l.indexOf) ? l.indexOf(r) > -1 : false;
+        if (l && isArray(l))
+            return l.some((i) => equal(i, r));
+        if (l && isString(l))
+            return l.indexOf(toValue(r)) > -1;
+        return false;
     },
     'not': (v, ctx) => isFalsy(toValue(v), ctx),
     'and': (l, r, ctx) => isTruthy(toValue(l), ctx) && isTruthy(toValue(r), ctx),
