diff --git a/node_modules/mjml-core/lib/index.js b/node_modules/mjml-core/lib/index.js
index 6bd5cb3..207e623 100644
--- a/node_modules/mjml-core/lib/index.js
+++ b/node_modules/mjml-core/lib/index.js
@@ -84,7 +84,7 @@ var _find2 = _interopRequireDefault(require("lodash/find"));
 var _path = _interopRequireDefault(require("path"));
 var _juice = _interopRequireDefault(require("juice"));
 var _jsBeautify = require("js-beautify");
-var _htmlMinifier = require("html-minifier");
+// var _htmlMinifier = require("html-minifier");
 var _cheerio = require("cheerio");
 var _mjmlParserXml = _interopRequireDefault(require("mjml-parser-xml"));
 var _mjmlValidator = _interopRequireWildcard(require("mjml-validator"));
