diff --git a/node_modules/redoc/bundles/redoc.lib.js b/node_modules/redoc/bundles/redoc.lib.js
index 28df6d7..681be08 100644
--- a/node_modules/redoc/bundles/redoc.lib.js
+++ b/node_modules/redoc/bundles/redoc.lib.js
@@ -9064,9 +9064,15 @@ var Schema_SchemaModel = /** @class */ (function () {
         this.isCircular = schema['x-circular-ref'];
         this.title = schema.title || isNamedDefinition(this.pointer) && JsonPointer_JsonPointer.baseName(this.pointer) || '';
         this.description = schema.description || '';
-        this.type = schema.type || detectType(schema);
+        if (Array.isArray(schema.type) && schema.type.includes("null")) {
+          this.nullable = true;
+          const other = schema.type.filter(t => t !== "null");
+          this.type = other.length === 1 ? other[0] : other;
+        } else  {
+          this.nulable = !!schema.nullable;
+          this.type = schema.type || detectType(schema);
+        }
         this.format = schema.format;
-        this.nullable = !!schema.nullable;
         this.enum = schema.enum || [];
         this.example = schema.example;
         this.deprecated = !!schema.deprecated;
