diff --git a/node_modules/slate-react/dist/plugin/with-react.d.ts b/node_modules/slate-react/dist/plugin/with-react.d.ts
index 865264c..2229e2d 100644
--- a/node_modules/slate-react/dist/plugin/with-react.d.ts
+++ b/node_modules/slate-react/dist/plugin/with-react.d.ts
@@ -1,4 +1,5 @@
 import { ReactEditor } from './react-editor';
+import { Editor } from "slate";
 /**
  * `withReact` adds React and DOM specific behaviors to the editor.
  *
@@ -7,5 +8,5 @@ import { ReactEditor } from './react-editor';
  *
  * See https://docs.slatejs.org/concepts/11-typescript to learn how.
  */
-export declare const withReact: <T extends ReactEditor>(editor: T) => T & ReactEditor;
+export declare const withReact: <T extends Editor>(editor: T) => T & ReactEditor;
 //# sourceMappingURL=with-react.d.ts.map
\ No newline at end of file
